gradle/libs.versions.toml

```
@@ -12,9 +12,20 @@ androidx-material = "1.12.0"
 androidx-test-junit = "1.1.5"
 compose-plugin = "1.6.10"
 junit = "4.13.2"
+kamelImage = "0.9.5"
+koin = "3.5.6"
 kotlin = "2.0.0"
+kotlinxCoroutinesCore = "1.9.0-RC"
+ktorClientCore = "2.3.11"
+voyagerNavigator = "1.0.0"
 
 [libraries]
+insert-koin-koin-core = { module = "io.insert-koin:koin-core" }
+kamel-image = { module = "media.kamel:kamel-image", version.ref = "kamelImage" }
+koin-bom = { module = "io.insert-koin:koin-bom", version.ref = "koin" }
+koin-android = { module = "io.insert-koin:koin-android", version.ref = "koin" }
+koin-androidx-compose = { module = "io.insert-koin:koin-androidx-compose", version.ref = "koin" }
+koin-core = { module = "io.insert-koin:koin-core", version.ref = "koin" }
```

Acho que não precisa referenciar as versões do koin nas outras dependências. O koin-bom já garante que eles peguem a dependencia correta. E o koin-core está inserido 2 vezes

```suggestion
kamel-image = { module = "media.kamel:kamel-image", version.ref = "kamelImage" }
koin-bom = { module = "io.insert-koin:koin-bom", version.ref = "koin" }
koin-android = { module = "io.insert-koin:koin-android" }
koin-androidx-compose = { module = "io.insert-koin:koin-androidx-compose" }
koin-core = { module = "io.insert-koin:koin-core" }
```